styles/AdvancedEditor.css

```diff
@@ -362,18 +362,19 @@ span.comment-marker {
   font-weight: 400 !important;
   line-height: 16px !important;
 }
-
-.ProseMirror .author-notes-container h1 {
+.ProseMirror h1.smallBoldHeader {
```

Comment:
is there a reason why "smallBoldHeader" class doesn't conform with selectors style that we normally use? I.e.:
small-bold-header? Can we rename it to reflect semantics and not internal properties of the class since I doubt we will ever reuse this class in other small-bold cases.

Reply:
renamed to .authors-info-header